lib/node_modules/@stdlib/array/base/mskbinary3d/package.json

```diff
@@ -0,0 +1,74 @@
+{
+    "name": "@stdlib/array/base/mskbinary3d",
```

Comment:
The indentation in this file is off. Looks like it just needs to be de-dented by 2 spaces.